python/ThirteenTeV/Higgs/Hadronizer_TuneCUETP8M1_13TeV_aMCatNLO_GluGluHToGG_M-120_LHE_pythia8_cff.py

```diff
@@ -0,0 +1,41 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.Pythia8CUEP8M1Settings_cfi import *
+from Configuration.Generator.Pythia8aMCatNLOSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8HadronizerFilter",
+    maxEventsToPrint = cms.untracked.int32(1),
+    pythiaPylistVerbosity = cms.untracked.int32(1),
+    filterEfficiency = cms.untracked.double(1.0),
+    pythiaHepMCVerbosity = cms.untracked.bool(False),
+    comEnergy = cms.double(13000.),
+    PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CUEP8M1SettingsBlock,
+        pythia8aMCatNLOSettingsBlock,
+        processParameters = cms.vstring(
+            'JetMatching:setMad = off',
+            'JetMatching:scheme = 1',
+            'JetMatching:merge = on',
+            'JetMatching:jetAlgorithm = 2',
+            'JetMatching:etaJetMax = 999.',
+            'JetMatching:coneRadius = 1.',
+            'JetMatching:slowJetPower = 1',
+            'JetMatching:qCut = 30.', #this is the actual merging scale
+            'JetMatching:doFxFx = on',
+            'JetMatching:qCutME = 10.',#this must match the ptj cut in the lhe generation step
```

Comment:
yes, as you can see from
https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/13TeV/higgs/ggh012j_5f_NLO_FXFX_125/ggh012j_5f_NLO_FXFX_125_run_card.dat

2015-06-02 13:14 GMT+02:00 Josh Bendavid <notifications@github.com>:

> In
> python/ThirteenTeV/Higgs/Hadronizer_TuneCUETP8M1_13TeV_aMCatNLO_GluGluHToGG_M-120_LHE_pythia8_cff.py
> <https://github.com/cms-sw/genproductions/pull/429#discussion_r31512316>:
>
> > +    comEnergy = cms.double(13000.),
> > +    PythiaParameters = cms.PSet(
> > +        pythia8CommonSettingsBlock,
> > +        pythia8CUEP8M1SettingsBlock,
> > +        pythia8aMCatNLOSettingsBlock,
> > +        processParameters = cms.vstring(
> > +            'JetMatching:setMad = off',
> > +            'JetMatching:scheme = 1',
> > +            'JetMatching:merge = on',
> > +            'JetMatching:jetAlgorithm = 2',
> > +            'JetMatching:etaJetMax = 999.',
> > +            'JetMatching:coneRadius = 1.',
> > +            'JetMatching:slowJetPower = 1',
> > +            'JetMatching:qCut = 30.', #this is the actual merging scale
> > +            'JetMatching:doFxFx = on',
> > +            'JetMatching:qCutME = 10.',#this must match the ptj cut in the lhe generation step
>
> Was ptj=10 what was used for the corresponding gridpacks?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/genproductions/pull/429/files#r31512316>.
>